refactor.rb

```diff
@@ -0,0 +1,15 @@
+class People
+	attr_reader :age, :phase
+	attr_accessor :name
+
+	def initialize(options={})
+		@phase = 3 # not sure 
```

Comment:
This is fine to put here.  Most people have phase 3 as the default so putting it here covers most cases, and you correctly override it after calling super in the `Student` class, so this is fine here.  I'd probably put it in a common superclass of `Teacher` and `ApprenticeTeacher` but you don't have that (instead opting for the `Salary` module, so given that choice, this placement makes sense.